liveblog.php

```diff
@@ -388,14 +389,14 @@ public static function is_liveblog_post( $post_id = null ) {
 	 * @return bool
 	 */
 	public static function is_viewing_liveblog_post() {
-		return (bool) ( is_single() && self::is_liveblog_post() );
+		return (bool) ( is_single() && self::is_liveblog_post() || is_page() && self::is_liveblog_post() );
```

Comment:
This conditional could probably be structured differently to avoid duplicating `self::is_liveblog_post()`